tests/test_filters.py

```diff
@@ -259,6 +260,10 @@ def test_issue_399():
     reader.pages[1].extract_text()
 
 
+@pytest.mark.skipif(
```

Comment:
I disagree with this. I've prepared a PR to fix this test.
Actually currently all tests are passing

Reply:
It should have been xfail and the reason was phrases bad, but I did see your pr :-)